libs/components/lists/src/lib/modules/repeater/repeater-item.component.ts

```diff
@@ -55,17 +55,16 @@ export class SkyRepeaterItemComponent
   implements OnDestroy, OnInit, AfterViewInit
 {
   /**
-   * Make the first, non-disabled item tab-focusable.
-   * - Disabled items should not be focusable per [W3C](https://www.w3.org/TR/wai-aria-practices-1.1/#kbd_disabled_controls).
-   * - One item per list/grid/listbox should be tab focusable per [W3C](https://www.w3.org/TR/wai-aria-practices-1.1/#grid).
+   * Make the first, non-disabled item tab-focusable in a selectable repeater.
+   * - Disabled items should not be focusable per [W3C](https://www.w3.org/WAI/ARIA/apg/practices/keyboard-interface/#focusabilityofdisabledcontrols).
+   * - One item per grid should be tab focusable per [W3C](https://www.w3.org/WAI/ARIA/apg/patterns/grid/).
    */
   public get tabindex(): 0 | -1 {
     return this.#repeaterService.items.filter((item) => !item.disabled)[0] ===
-      this
+      this && this.selectable
```

Comment:
This operation can be cleaned up a bit. Put the `selectable` check in the same place as the `disabled` check, and instead of using `filter(...)[0]`, use `find()`. (I don't think `.find()` existed when this component was first written.)
```
public get tabindex(): 0 | -1 {
  return this.#repeaterService.items.find(
    (item) => !item.disabled && this.selectable,
  ) === this
    ? 0
    : -1;
}
```